tests/Feature/Events/WordpressEventPushTest.php

```diff
@@ -27,6 +27,11 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        // Skip all tests if WordPress integration is disabled
+        if (config('restarters.features.wordpress_integration') === false) {
+            $this->markTestSkipped('WordPress integration is disabled.');
+        }
```

Comment:
We should move this above `parent::setUp()`.

Reply:
Same in `tests/Feature/Groups/WordpressGroupPushTest.php`.

Reply:
Ha so I just tried applying that change and looks like we do need it after the `parent::setUp` call. Otherwise, the `config` is not setup:

![image](https://github.com/user-attachments/assets/3d9b4731-6e36-4111-b363-7109323e28fb)

Reply:
Wild, okay 👍